src/dialog.tsx

```diff
@@ -156,16 +158,42 @@ export const DialogContent = (props: any): JSX.Element => {
                 setDeployErrorMessage(errorMsg)
             }
             else if (result?.detail || result?.message) {
-
                 errorMsg.detail = result.detail || result.message
                 setDeployErrorMessage(errorMsg)
-            } else {
+                }
+            else {
                 setDeploymentURL(DEPLOYMENT_ENDPOINTS.NEW_JOB + result?.project_id + "/" + result?.id)
                 props?.metadata?.set("ploomber", { "project_id": result?.project_id })
                 props?.context?.save()
             }
+        })
+        }
+        else {
+            await requestAPI<any>('projects', {
+            body: JSON.stringify({'api_key': APIKey, 'project_id': projectId }),
+            method: 'POST'
+        }).then(reply => {
+            var result = reply["project_details"]
+            var errorMsg: {
+                type: string,
+                detail: any
+            } = {
+                type: "generic",
+                detail: ""
+            }
+            console.log(result)
```

Comment:
remove this console.log

Reply:
I removed  the entire code block as it was unnecessary

Reply:
don't forget to include a link to the changes

Reply:
Found the commit: https://github.com/ploomber/jupysql-plugin/pull/90/commits/ccc584b2968e20daa1164de1c14ae055653800a6c